### DISCUSSION TITLE
v0.8

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#575] [New Releases]
## What's Changed
* Allow overriding shortcut keys by @dail8859 in https://github.com/dail8859/NotepadNext/pull/571
* Allow setting default font and size by @dail8859 in https://github.com/dail8859/NotepadNext/pull/552
* Allow specifying custom tab bar context menu
* Add match count to Quick Find by @dail8859 in https://github.com/dail8859/NotepadNext/pull/572
* Add shortcuts to switch to next or previous tab
* Add folding shortcuts
* Close the initial editor when opening a file
* Add 'Exit on last tab closed' option
* Add `-n` command line parameter to open file at specified line number
* Add Polish translation by @kocio-pl in https://github.com/dail8859/NotepadNext/pull/519
* Add Russian translation by @dchapyshev and @adem4ik in https://github.com/dail8859/NotepadNext/pull/521 and https://github.com/dail8859/NotepadNext/pull/559
* Update Portuguese translation by @hugok79 in https://github.com/dail8859/NotepadNext/pull/563
* Check filename before save on Save a Copy As... by @detinho in https://github.com/dail8859/NotepadNext/pull/527
* Update CFBundleIdentifier by @FnControlOption in https://github.com/dail8859/NotepadNext/pull/529
* Adapt the Find/Replace dialog UI for MAC by @ErickWei in https://github.com/dail8859/NotepadNext/pull/538
* Adapt the Preference dialog UI for MAC  by @ErickWei in https://github.com/dail8859/NotepadNext/pull/542
* Update Linux screenshot by @japanoise in https://github.com/dail8859/NotepadNext/pull/549
* Restore status bar visibility on startup

## New Contributors
* @kocio-pl made their first contribution in https://github.com/dail8859/NotepadNext/pull/519
* @dchapyshev made their first contribution in https://github.com/dail8859/NotepadNext/pull/521
* @Gitoffthelawn made their first contribution in https://github.com/dail8859/NotepadNext/pull/524
* @detinho made their first contribution in https://github.com/dail8859/NotepadNext/pull/527
* @FnControlOption made their first contribution in https://github.com/dail8859/NotepadNext/pull/529
* @ErickWei made their first contribution in https://github.com/dail8859/NotepadNext/pull/538
* @japanoise made their first contribution in https://github.com/dail8859/NotepadNext/pull/549
* @adem4ik made their first contribution in https://github.com/dail8859/NotepadNext/pull/559
* @hugok79 made their first contribution in https://github.com/dail8859/NotepadNext/pull/563

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.7...v0.8

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.8'>v0.8</a>.</em>

**Comment by BBQigniter** [comment]
Hello,

since I updated Notepad Next (using Flatpak version on Fedora 40 with KDE Plasma + Wayland) I do not see the top menu bar anymore, is there some option to enable it or is this a bug? I also tested this with the AppImage.

![image](https://github.com/dail8859/NotepadNext/assets/8897660/40b9e029-43ec-4765-9abf-6aa2dd1f7df4)

As you can see in the screenshot there is no "File", "Edit", ... "Help" menu anymore.

Thanks for help in advance.

**Reply by dail8859** [reply]
So there is a setting to hide the menu bar, but even testing it now on Windows seems to have issues with it not persisting. It always defaults to showing it.

But if it is hidden on start up there definitely seems to be some kind of bug. Probably specific to your distro/setup but not sure.

If you start the application with `--reset-settings` and will restore all the settings to defaults. Keep in mind this might delete any temporary files if it is set to restore them, so save those first.

**Reply by BBQigniter** [reply]
Thank you very much for the tip, i tried with `--reset-settings` but unfortunately it didn't help. 

After trying and googling around a little bit more this seems to be some weird bug between Qt and Wayland as far as I understood. I also selfcompiled NotepadNext on my Fedora but still I was missing the "Global menu bar". But finally I found this post in a thread where something similar was discussed https://forum.garudalinux.org/t/incorrect-behavior-of-some-programs-using-qt-on-kde-6-wayland/35720/14 

Before I used the standard Fedora Theme - I switched to Breeze and now it works :)

**Reply by dail8859** [reply]
Interesting. Appreciate the time you put into this to track it down. Doesn't appear there's anything to address this on the application end but good to know this isn't just an isolated issue with this application.

**Comment by CodeXBlockers** [comment]
[Screen recording 2024-10-11 12.13.27 AM.webm](https://github.com/user-attachments/assets/ec76875b-e508-44e1-be0c-2a570f048da0)
hello everyone, does anyone know what is the root cause of this problem? when I enter the app itself, it lets me in for a sec but immediately exit afterwards, any help is appreciated, thanks in advance

**Reply by dail8859** [reply]
Does it only do it when you use the language menu? Also (not sure what OS you are using) you could try running it from command line. It does have debug log but depending on OS and installation method may not print it to the console but worth a try.
